fem/conduitdatacollection.cpp

```diff
@@ -939,6 +940,28 @@ ConduitDataCollection::GridFunctionToBlueprintField(mfem::GridFunction *gf,
 
 }
 
+//---------------------------------------------------------------------------//
+bool
+ConduitDataCollection::HasBoundaryElements(mfem::Mesh* mesh)
```

Comment:
No strong opinion, but this seems useful in general, so it could be moved as a method of `ParMesh`.

Reply:
Should it be a virtual method of `mfem::Mesh` with an override in `ParMesh`?

Reply:
@aaronskinner, I can make the move to an `Mesh`/`ParMesh` method for you if you prefer

Reply:
Yes, I think it should move; these seems like a useful public method for that class.  I'm less sure about the other functions that get the boundary element type, because here we're assuming the type is the same for all boundary elements on the mesh.  I'm now sure how the blueprint even looks if they're different...  I'm also not sure it matters based on Cyrus' later comment.